integration-tests/maven/src/test/java/io/quarkus/maven/it/DevMojoIT.java

```diff
@@ -838,15 +840,15 @@ public void testThatNewResourcesAreServed() throws MavenInvocationException, IOE
         await()
                 .pollDelay(100, TimeUnit.MILLISECONDS)
                 .atMost(1, TimeUnit.MINUTES)
-                .until(() -> DevModeTestUtils.getHttpResponse("/lorem.txt").contains("Lorem ipsum"));
+                .until(() -> DevModeTestUtils.getHttpResponse("/lorem.txt"), containsString("Lorem ipsum"));
 
         // Update the resource
         String uuid = UUID.randomUUID().toString();
         FileUtils.write(source, uuid, "UTF-8");
         await()
                 .pollDelay(100, TimeUnit.MILLISECONDS)
                 .atMost(1, TimeUnit.MINUTES)
-                .until(() -> DevModeTestUtils.getHttpResponse("/lorem.txt").contains(uuid));
+                .until(() -> DevModeTestUtils.getHttpResponse("/lorem.txt"), equalTo(uuid));
```

Comment:
You changed the meaning here. That's what you wanted to do?

Reply:
YEs